tools/test-static-code

```diff
@@ -50,6 +50,11 @@ if out=$(find src/ pkg/ -name '*.js' -o -name '*.jsx' | xargs grep -n -E "(gette
     echo "$out" >&2
     echo "not ok 3 js-translatable-strings"
     fail=1
+elif out=$(grep -r 'translatable=["'\'']yes' pkg/ doc/); then
```

Comment:
I would not limit to to `=yes`, I think that `grep -r 'translatable='` would be enough, but just nitpick :)

Reply:
Since I fixed all occurrences let's keep it like this so that there is some consistency regarding how we use it.